lib/id3c/cli/redcap.py

```diff
@@ -69,6 +70,18 @@ def instruments(self) -> List[str]:
         return self._instruments
 
 
+    @property
+    def events(self) -> List[str]:
+        """
+        Names of all events in this REDCap project.
+        """
+        if not self._events:
+            nameof = itemgetter("unique_event_name")
+            self._events = list(map(nameof, self._fetch("event")))
```

Comment:
Not all REDCap projects have events, so I'm not sure if we need some sort of special handling here.

Reply:
Using the REDCap API Playground for the Swab & Send project, it looks like this API request returns a default value `event_1_arm_1` for `unique_event_name`. So maybe it's not a problem?

Reply:
@joverlee521 Your first intuition is correct. Many (maybe most?) of our REDCap projects are "longitudinal" projects, which means they define events. "Classic" projects, however, do not use events and the `content=event` API call will return an error:

    HTTP/1.1 400 Bad Request
    {"error":"You cannot export events for classic projects"}

This setting is determined by the project's "Use longitudinal data collection with defined events?" checkbox on the project setup page. There's more documentation about how classic vs. longitudinal projects and how they interact with repeatable instruments/events on [this doc page](https://redcap.iths.org/redcap_v9.9.0/ProjectSetup/repeating_instruments_events_info.php).

Reply:
Thanks. It's not clear to me whether a classic REDCap project would silently ignore an `events` parameter in the data payload or throw an HTTP error. I'm also not sure how to test this, as seemingly every REDCap project of ours is longitudinal, not classic.

Reply:
The project we use to coordinate clinical data pulls from UW is "classic": https://redcap.iths.org/redcap_v9.9.0/index.php?pid=19915

Reply:
(which is the project I used to see that making an API request with `content=event` fails)

Reply:
Thanks!